lms/djangoapps/django_comment_client/utils.py

```diff
@@ -221,7 +221,7 @@ def _filter_unstarted_categories(category_map):
 
         for child in unfiltered_map["children"]:
             if child in unfiltered_map["entries"]:
-                if unfiltered_map["entries"][child]["start_date"] <= now:
+                if unfiltered_map["entries"][child]["start_date"] <= now or course.self_paced:
```

Comment:
Pushed. I was having second thoughts about the change and wanted to think a little harder before going with it.